### DISCUSSION TITLE
LinkTap

### DISCUSSION THREAD

**Discussion by dasfuu** [Koenkk/zigbee2mqtt#6333] [Devices]
Hello,

Can anyone help with this device? It says it uses a "proprietary Zigbee protocol". But I think it should still be possible to support in zigbee2mqtt.

Just leaving the amazon link here for reference:
https://www.amazon.com/LinkTap-Wireless-Controlled-Coverage-Installation/dp/B07BKZ3YZY

There is a G2 version too with a flow meter attached.

**Comment by calvinbui** [comment]
did you ever find out?

@shanelord01 i saw you had a linktap device too over in https://github.com/Koenkk/zigbee2mqtt/issues/13800
